src/core/lib/iomgr/endpoint_pair_windows.cc

```diff
@@ -63,8 +63,16 @@ static void create_sockets(SOCKET sv[2]) {
   GPR_ASSERT(svr_sock != INVALID_SOCKET);
 
   closesocket(lst_sock);
-  grpc_tcp_prepare_socket(cli_sock);
-  grpc_tcp_prepare_socket(svr_sock);
+  grpc_error_handle error = grpc_tcp_prepare_socket(cli_sock);
+  if (!error.ok()) {
+    gpr_log(GPR_ERROR, "Prepare cli_sock failed with error: %s",
```

Comment:
Seems like some tests are expecting no error logging. We can change `GPR_ERROR` to `GPR_INFO`. Same below.

Sorry for the back-and-forth!

Reply:
I reduced it to WARN for now, hopefully that's good enough